wordpress-mobile/WordPress-iOS
# Adds the option to display posts in a compact way

Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/11646

This PRs adds an option to see the post list in a more compact way:

<img src="https://user-images.githubusercontent.com/7040243/58523483-f8c33480-819a-11e9-9ce2-d9049669c9fa.gif" width="300" />

And also:

* Always displays the posts cells based on the user choice (compact or card);
* Shows a compact ghost cell if the user chooses the compact view;
* When seeing compact cells, the action sheet always shows a "View" option

To test:
- Check if the ghost cells appear when loading fresh content;
- Check if your preference is saved: if you select compact cells it should always display compact cells unless you change it - the default is the comfortable cell;
- Check if a "View" option is shown in the action sheet for compact cells - but not for the card cell;
